apps/web/src/assets/scss/card.scss

```diff
@@ -139,8 +139,8 @@
       }
 
       .fr-card__title {
-        @if meta.type-of($properties) == map and map.has-key($properties, "is-inverted") and map.get($properties, "is-inverted") == true {
-          color: map.get($properties, "color");
+        @if meta.type-of($properties) == map {
+          color: map.get($properties, "color-with-background");
```

Comment:
praise: super ! 
Par contre, on doit pouvoir supprimer la propriété `is-inverted` des la liste des couleurs

Reply:
done pour le remove de is-inverted